quickannotator/api/v1/image/utils.py

```diff
@@ -0,0 +1,26 @@
+from quickannotator.db import db_session
+import quickannotator.db.models as models
+import large_image
```

Comment:
where'd large image come into this?

Reply:
I think we previously discussed this - I chose large image for serving WSI tiles / reading regions because
1. it wraps openslide while also adding guardrails logic for slides with missing levels and metadata - these were problems I ran into while trying out the openslide tileserver example.
2. it is modular, e.g., we can pip install large_image[bioformats] etc., to easily add support for formats which openslide does not support
3. package overhead was relatively low - I forget how much but I think it was less than 20mb
4. It was the tile-serving library of choice for geojs and DSA and I was able to quickly get the tile server up and running in just a few lines of code.

Reply:
I guess it follows that we should use large_image throughout the system - I can see switching from openslide to large_image is a little inconvenient given that the interface is a little different (`getRegion()` has different params vs `read_region()`).